CreateTheTables.sql

```diff
@@ -0,0 +1,65 @@
+CREATE TABLE tblEnemy(
```

Comment:
There is a small and tricky sentence you missed here at the beginning, Could you guess what it is?